workers-site/redirects/oldDocs.js

```diff
@@ -18,12 +19,12 @@ export const oldDocsMap = new Map([
   ['/api/config-api-for-enterprise', '/tooling/api'],
   ['/api/resource-bindings', '/tooling/api/bindings'],
   ['/api/resource-bindings/kv-namespaces', '/reference/storage/api'],
-  ['/api/resource-bindings/webassembly-modules', ''],
-  ['/api/route-matching', ''],
+  ['/api/resource-bindings/webassembly-modules', '/tooling/api/bindings'],
+  ['/api/route-matching', '/about/routes'],
   ['/deploying-workers', '/tooling'],
-  ['/deploying-workers/github-action', ''],
+  ['/deploying-workers/github-action', '/tooling/integrations'],
   ['/deploying-workers/serverless', '/tooling/serverless'],
-  ['/deploying-workers/terraform', ''],
+  ['/deploying-workers/terraform', ''], // TODO replace this with terraform link in issue https://github.com/cloudflare/workers-docs/issues/253
```

Comment:
there might be a git conflict here, i think keeping this line the same (no comment) will help it merge easier